runtime/Cargo.toml

```diff
@@ -7,6 +7,7 @@ repository = "https://github.com/solana-labs/solana"
 license = "Apache-2.0"
 homepage = "https://solana.com/"
 edition = "2018"
+build = "../sdk/build.rs"
```

Comment:
Due to  the same restriction of [this](https://github.com/solana-labs/solana/pull/8012/files#r374398085), we're also forced to add this magic to use frozen_abi in addition to the usual of adding to `[dependencies]`.